.github/workflows/ci.yml

```diff
@@ -72,14 +73,25 @@ jobs:
           conda-solver: libmamba
           channels: conda-forge,bioconda
 
-      - name: Run pipeline with test data (singularity)
+      - name: Set up Conda
+        if: matrix.profile == 'conda'
         run: |
           echo $(realpath $CONDA)/condabin >> $GITHUB_PATH
           echo $(realpath python) >> $GITHUB_PATH
 
       - name: Clean up Disk space
         uses: jlumbroso/free-disk-space@54081f138730dfa15788a46383842cd2f914a1be # v1.3.1
 
+      - name: "Run pipeline with test data ${{ matrix.NXF_VER }} | ${{ matrix.test_name }} | ${{ matrix.profile }}"
+        continue-on-error: ${{ matrix.NXF_VER == 'latest-everything' }}
+        run: |
+          nextflow run ${GITHUB_WORKSPACE} -profile test,docker --outdir ./results
+
+      - name: Run pipeline with test data (singularity)
+        run: |
+          nextflow run ${GITHUB_WORKSPACE} -profile test,singularity --outdir ./results
+        if: "${{ github.base_ref == 'master' }}"
+
```

Comment:
looks ok, but why was it not run with docker and singularity before? just wandering if there was a justified reason for this and we should leave it as is